src/PostgREST/Config.hs

```diff
@@ -20,20 +20,20 @@ Other hardcoded options such as the minimum version number also belong here.
 {-# LANGUAGE TemplateHaskell       #-}
 {-# OPTIONS_GHC -fno-warn-type-defaults #-}
 
-module PostgREST.Config ( prettyVersion
-                        , docsVersion
-                        , CLI (..)
-                        , Command (..)
-                        , AppConfig (..)
-                        , configDbPoolTimeout'
-                        , dumpAppConfig
-                        , Environment
-                        , readCLIShowHelp
-                        , readEnvironment
-                        , readValidateConfig
-                        )
-       where
-
+module PostgREST.Config
+  ( AppConfig (..)
+  , Environment
+  , configDbPoolTimeout'
+  , docsVersion
+  , dumpAppConfig
+  , parseSecret
+  , prettyVersion
+  , readValidateConfig
+  ) where
+
+import qualified Crypto.JOSE.Types      as JOSE
+import qualified Crypto.JWT             as JWT
+import qualified Data.Aeson             as JSON
 import qualified Data.ByteString        as B
 import qualified Data.ByteString.Base64 as B64
 import qualified Data.ByteString.Char8  as BS
```

Comment:
Those seem like they'd need some kind standardization. Are we using `Char8` anywhere else? I suggest we use `Data.ByteString as BS` - and then `BS64` and `BS8` or something?

Reply:
Used BS8 for now